app/src/assets/labware/getLabware.ts

```diff
@@ -7,50 +6,50 @@ import type {
 
 // require all definitions in the labware/definitions/1 directory
 // require.context is webpack-specific method
-const labwareSchemaV1DefsContext = (require: any).context(
+const labwareSchemaV1DefsContext = (require as any).context(
   '@opentrons/shared-data/labware/definitions/1',
   true, // traverse subdirectories
   /\.json$/, // import filter
   'sync' // load every definition into one synchronous chunk
 )
-let labwareSchemaV1Defs: $ReadOnlyArray<LabwareDefinition1> | null = null
-function getLegacyLabwareDefs(): $ReadOnlyArray<LabwareDefinition1> {
+let labwareSchemaV1Defs: Readonly<LabwareDefinition1[]> | null = null
+function getLegacyLabwareDefs(): Readonly<LabwareDefinition1[]> | null {
   if (!labwareSchemaV1Defs) {
     labwareSchemaV1Defs = labwareSchemaV1DefsContext
       .keys()
-      .map(name => labwareSchemaV1DefsContext(name))
+      .map((name: string) => labwareSchemaV1DefsContext(name))
   }
 
   return labwareSchemaV1Defs
 }
 
 export function getLegacyLabwareDef(
-  loadName: ?string
+  loadName: string | null | undefined
 ): LabwareDefinition1 | null {
-  const def = getLegacyLabwareDefs().find(d => d.metadata.name === loadName)
+  const def = getLegacyLabwareDefs()?.find(d => d.metadata.name === loadName)
```

Comment:
Related to comment above, `getLegacyLabwareDefs()` will never be null

```suggestion
  const def = getLegacyLabwareDefs().find(d => d.metadata.name === loadName)
```